### PR TITLE
Adds inRange package

### PR DESCRIPTION
**Pull request by EvandroLG**
This PR is adding the inRange package, which was requested in issue #410 :)

### CODE REVIEW

**Comment by angus-c** [comment]
Thanks @EvandroLG, great job! Just a few minor nits.

**Comment by EvandroLG** [comment]
Which ones exactly @angus-c ? :D

**Review by angus-c — CHANGES_REQUESTED** [review]

**Comment by angus-c** [comment]
> Which ones exactly @angus-c ? :D

Whoops I had forgotten to hit submit review! Apologies, should be there now

File: packages/number-in-range/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export default function inRange(
+  number: number,
+  start: number,
+  end?: number
```
[the remark this answers is not in the document]
Comment (EvandroLG) [reply packages/number-in-range/index.d.ts:4]: In TypeScript a required parameter can not follow an optional one :)

[the remark this answers is not in the document]
Comment (angus-c) [reply packages/number-in-range/index.d.ts:4]: Ok that's fair. I'm not a TS expert :)

**Pull request merged**